model/job.go

```diff
@@ -98,3 +100,47 @@ func (j *Job) Validate(c echo.Context) error {
 	}
 	return nil
 }
+
+func (j *Job) tag(db *extensions.PGClient, name, menssage, state string) {
+	status := &Status{
+		Name:      name,
+		JobID:     j.ID,
+		ID:        uuid.NewV4(),
+		CreatedAt: time.Now().UnixNano(),
+	}
+	_, err := db.DB.Model(status).OnConflict("DO NOTHING").Insert()
+	if err != nil {
+		panic(err)
+	}
+	status = &Status{}
+	err = db.DB.Model(status).Where("job_id = ? AND name = ?", j.ID, name).First()
```

Comment:
Why are you fetching the status you've just inserted into the db?

Reply:
During the insertion one conflict can happen, in this case, the fetch action is needed to know the existing status id.